extension.js

```diff
@@ -37,9 +38,9 @@ function launch_extension_prefs(uuid) {
         global.create_app_launch_context(timestamp, -1)
     );
 }
-const WebSearchDialog = class WebSearchDialog extends ModalDialog.ModalDialog {
-    constructor() {
-        super({
+const WebSearchDialog = GObject.registerClass (class WebSearchDialog extends ModalDialog.ModalDialog {
```

Comment:
This fails for me on 3.32.2: `GObject.registerClass used with invalid base class (is ModalDialog)`

Reply:
Sounds like this post : https://github.com/ubuntu/gnome-shell-extension-appindicator/issues/161

3.34 seems not compatible with previous versions